src/model/block/ISetContactPropertyBlockConfig.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Flow Interoperability Project (flowinterop.org)
+ * Flow Runner
+ * Copyright (c) 2019, 2020 Viamo Inc.
+ * Authored by: Brett Zabos (brett.zabos@viamo.io)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+ * documentation files (the "Software"), to deal in the Software without restriction, including without limitation the
+ * rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of the
+ * Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+ * WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+ * OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ **/
+import {SetContactProperty} from './IBlockConfig'
+
+export interface ISetContactPropertyBlockConfig {
+  set_contact_property: SetContactProperty
+}
```

Comment:
Seeing that this was done, as this new case requires this to be set, use something like this instead, which requires a much smaller changeset AND requires much much much less verbosity when working with BlockConfigs from here on:
```

export interface ISetContactPropertyBlockConfig {
  set_contact_property?: SetContactProperty
}

export interface ISetContactPropertyBlockConfigRequired extends ISetContactPropertyBlockConfig {
  set_contact_property: SetContactProperty
}
```

Reply:
fixed with the above task
> improve ISetContactPropertyBlock interfaces declaration